jablib/src/main/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheck.java

```diff
@@ -9,16 +9,47 @@
 import java.util.SequencedCollection;
 import java.util.Set;
 import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import org.jabref.logic.bibtex.comparator.BibEntryByCitationKeyComparator;
 import org.jabref.logic.bibtex.comparator.BibEntryByFieldsComparator;
 import org.jabref.logic.bibtex.comparator.FieldComparatorStack;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.SpecialField;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.field.UserSpecificCommentField;
 import org.jabref.model.entry.types.EntryType;
 
 public class BibliographyConsistencyCheck {
 
+    private static final Set<Field> FILTERED_FIELDS = Stream
+            .concat(
+                    StandardField.AUTOMATIC_FIELDS.stream(),
+                    Stream.of(
+                            StandardField.COMMENT,
+                            StandardField.CROSSREF,
+                            StandardField.CITES,
+                            StandardField.PDF,
+                            StandardField.REVIEW,
+                            StandardField.SORTKEY,
+                            StandardField.SORTNAME,
+                            StandardField.TYPE,
+                            StandardField.XREF,
+                            StandardField.GROUPS
+                    )
+            )
+            .collect(Collectors.toSet());
+
+    private static Set<Field> filterExcludedFields(Collection<Field> fields) {
+        return fields.stream()
+                     .filter(field -> !FILTERED_FIELDS.contains(field))
```

Comment:
Inline the variable. You have multiple filterings below; and these are also filtered.

Reply:
at [`c613b02` (#13305)](https://github.com/JabRef/jabref/pull/13305/commits/c613b0295a5251aa2e46e891edbf42e9fd6d5ccc) @calixtus and me worked on (now)